src/util/triplea/mapXmlCreator/MapXmlCreator.java

```diff
@@ -303,7 +303,6 @@ public MapXmlCreator(final boolean testMode) {
       try {
         loadXmlFromFilePath(myFile.getAbsolutePath());
       } catch (SAXException | IOException | ParserConfigurationException e) {
-        // TODO Auto-generated catch block
         log(Level.SEVERE, "Default Map XML File could not be loaded from '" + myFile.getAbsolutePath() + "'.");
```

Comment:
Missing ERROR level is signfiicant. Wrong or custom logger being used.  Looks like the latter,  there is a static `Logger` object out there:
```
   final private static Logger logger = Logger.getLogger(MAP_XML_CREATOR_LOGGER_NAME);
 
   public static Logger getLogger() {
     return logger;
   }
```
All this should go away. Just use a standard SFL4J or log4j object, should be a number of examples in the non map creator code.